src/Microsoft.ML.Core/Data/RoleMappedSchema.cs

```diff
@@ -9,8 +9,10 @@
 namespace Microsoft.ML.Runtime.Data
 {
     /// <summary>
-    /// This contains information about a column in an IDataView. It is essentially a convenience
-    /// cache containing the name, column index, and column type for the column.
+    /// This contains information about a column in an <see cref="IDataView"/>. It is essentially a convenience cache
+    /// containing the name, column index, and column type for the column. The intended usage is that users of <see cref="RoleMappedSchema"/>
+    /// will have a convenient method of getting the index and type without having to separately query it through the <see cref="ISchema"/>,
+    /// since practically the first thing a consumer of a <see cref="RoleMappedSchema"/> will want to do once they get a mappping is
```

Comment:
I feel comment/summary is not complete here as sentence is ending in `is`?

Reply:
Sure. You go first then I'll adjust. This work in some ways will be helped by what you're doing as well, as you have noted.

Reply:
Hmmm I do that sometimes don't I.

---
In reply to: [198998270](https://github.com/dotnet/machinelearning/pull/446#discussion_r198998270) [](ancestors = 198998270)